src/QMCTools/qmcfinitesize.cpp

```diff
@@ -73,12 +73,18 @@ int main(int argc, char** argv)
       skparser = new SkParserScalarDat();
       skparser->parse(anxt);
     }
+    else if (a == "--hdf5")
+    {
+      skparser = new SkParserHDF5();
```

Comment:
Please use unique_ptr and make_unique instead of raw pointer and new.

Reply:
done